packages/java/src/components/MavenProject.tsx

```diff
@@ -23,129 +30,257 @@ export function PomFile({ projectConfig }: PomFileProps) {
   // The following spaghetti mess is so the final output is formatted correctly.
   // Needs to be this way for when some options aren't provided, there's no random whitespace,
   // and so that looped sections are indented correctly.
-  return <SourceFile path="pom.xml" filetype="xml">
+  return (
+    <SourceFile path="pom.xml" filetype="xml">
+      <PomFileRoot>
+        <hbr />
+        <List
+          joiner={
+            <>
+              <hbr />
+              <hbr />
```

Comment:
What's `hbr` and why do we need it twice here?

Reply:
Hardbreak (essentially, "\n\n"). I don't know except that the test expects it.